docs/latex/Inhalt/03_Inhalt/03_clean_architecture.tex

```diff
@@ -0,0 +1,61 @@
+\chapter{Clean Architecture}
+In diesem Kapitel wird die Architektur der entwickelten Software beschrieben. Diese wurde nach den in der Vorlesung besprochenen Prinzipien der Clean Architecture aufgebaut. Jeder der nachfolgenden Abschnitte repräsentiert dabei eine Schicht. In der Implementierung entspricht den Schichten 1, 2 und 3 jeweils ein Java-Module. Lediglich Schicht 0, welche die Plugins und das Main-Module mit der Main-Klasse der Applikation enthält, bildet eine Ausnahme: Hier entspricht jedem Plugin sowie dem Main-Module eine separates Java-Module, da diese komplett und leicht austauschbar sein sollen.
+
+Die Clean Architecture hält neben den vier für dieses Projekt verwendeten Schichten noch eine fünfte bereit, den Abstraction Code. Auf diese Schicht, wurde für dieses Projekt jedoch verzichtet, da für die in der Domäne behandelten Themengebiete \enquote{Kochrezepte} und \enquote{Lebensmittel} kein domänenübergreifendes Wissen notwendig war, welches Teil dieser Schicht hätte sein müssen. 
+
+\section{Schicht 3: Domain}
+Der Domain Code der Software wurde in dieser Schicht implementiert und befindet sich im Module \code{3-quickie-domain}, welches als solches frei von jeglichen Dependencies ist.
+
+Sie enthält als zentrale Schicht der Clean Architecture alle Entities und Value Objects, welche die Enterprise Business Logik der Software implementieren sowie die Interface, für die zugehörigen Repositories. Ihr genauer Inhalt ist in \autoref{fig:class-diag-domain} dargestellt.
```

Comment:
"Enterprise Business Logik" => "Business Logic"?

Reply:
"sowie die Interface" => "sowie die Interfaces"

Reply:
"die Interface, " => Komma weg?

Reply:
Enterprise Buiness Logic wird in den Folien explizit als Inhalt der Domain genannt in Abgrenzung zur Application Business Logic in der Application Layer. Den Rest habe ich umgesetzt.